microbus.gemspec

```diff
@@ -20,6 +20,8 @@ Gem::Specification.new do |s|
 
   s.add_development_dependency 'aruba', '~> 0.13'
   s.add_development_dependency 'cucumber'
+  # Cucumber depends on Cabin which apparently depends on this.
+  s.add_development_dependency 'minitest'
```

Comment:
Why do we need to add it if we don't depend on it? Shouldn't bundler resolve this dependency?

Reply:
There was a require 'minitest/spec' in there somewhere. Possibly and upstream bug, I just didn't take time to dive into it.

Reply:
```
vendor/bundle/gems/cabin-0.9.0/test/test_helper.rb
2:require "support/minitest-patch"
```

```
vendor/bundle/gems/cabin-0.9.0/test/support/minitest-patch.rb
2:require "minitest/spec"
```

Reply:
This def. doesn't seem right.

Reply:
Removed.